spec/models/concerns/reviewable_action_builder_spec.rb

```diff
@@ -0,0 +1,66 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+
```

Comment:
```suggestion
```

Small note. This line is not necessary as it is automatically included by https://github.com/discourse/discourse/blob/e6f9cde35e3f68d4ffb0a46a8361ce099a558ac6/.rspec#L1